lib/modeToPrimitiveType.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+var Cesium = require('cesium');
+var PrimitiveType = Cesium.PrimitiveType;
+var WebGLConstants = Cesium.WebGLConstants;
+
+module.exports = modeToPrimitiveType;
+
+function modeToPrimitiveType(mode) {
```

Comment:
Do we need this function?  Aren't the `WebGLConstants` and `PrimitiveType` values the same?

See https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/PrimitiveType.js